notebooks/Example_Notebook.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Ok so that's cool. It does highlight the changes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>